astro/src/styles/colors-and-themes.scss

```diff
@@ -96,11 +96,15 @@ $info:                              #066F80; // branded cyan
 // Secondary Subtle Theme
 $info-text-emphasis:                #004F4F; // cyan-80
 $info-bg-subtle:                    #67DCE0; // cyan-00  
-$info-border-subtle:                #33CCCC; // cyan-10
+$info-border-subtle:                #004F4F; // cyan-80
```

Comment:
You're changing the definition for what every info-subtle border.  Are we sure we want to do this instead of doing this a different way?